src/iasm.c

```diff
@@ -505,34 +508,42 @@ static PTRNTAB asm_classify(OP *pop, OPND *popnd1, OPND *popnd2,
     // How many arguments are there?  the parser is strictly left to right
     // so this should work.
 
-    if (!popnd1)
+    if (popnd1 == emptyOPND)
     {
         usNumops = 0;
     }
     else
     {
         popnd1->usFlags = opflags1 = asm_determine_operand_flags(popnd1);
-        if (!popnd2)
+        if (!opflags1)
```

Comment:
Why can't the opflags be 0? That's somewhat uncommon for flags and warrants more explicit code, e.g. if (opflags == OPFLAGSerror) or so.